### PR TITLE
services: ssdp includes

### PR DESCRIPTION
**Pull request by marcosfrm**
About...
https://github.com/firewalld/firewalld/blob/c11510e065fdbaeb8ff20f32e2628bdd70d30dd7/config/services/upnp-client.xml

can a service only include other?

### CODE REVIEW

File: config/services/ssdp.xml [hunk 1]
```diff
@@ -3,5 +3,5 @@
   <short>Simple Service Discovery Protocol (SSDP)</short>
   <description>The Simple Service Discovery Protocol (SSDP) is a network protocol based on the Internet protocol suite for advertisement and discovery of network services and presence information.</description>
   <port protocol="udp" port="1900"/>
-  <destination ipv4="239.255.255.250" ipv6="FF02::C"/>
+  <destination ipv4="239.255.255.250" ipv6="ff02::c"/>
```
Comment (erig0) [inline config/services/ssdp.xml:6]: Was this change intentional? Seems unrelated. Please remove.

**Review by erig0 — CHANGES_REQUESTED** [review]
> can a service only include other?

I'm not sure what you're asking. Services may include other services.

**Comment by marcosfrm** [comment]
> > can a service only include other?
> 
> I'm not sure what you're asking. Services may include other services.

I mean "UPnP Client" from a firewall POV is "UPnP discovery protocol", i.e. SSDP... hence can `upnp-client.xml` _only_ include ssdp?

**Comment by marcosfrm** [comment]
https://github.com/firewalld/firewalld/blob/c11510e065fdbaeb8ff20f32e2628bdd70d30dd7/config/services/upnp-client.xml#L5

btw this is wrong, because the source port may be random for inbound traffic (M-SEARCH, NOTIFY).

**Comment by erig0** [comment]
> > > can a service only include other?
> > 
> > 
> > I'm not sure what you're asking. Services may include other services.
> 
> I mean "UPnP Client" from a firewall POV is "UPnP discovery protocol", i.e. SSDP... hence can `upnp-client.xml` _only_ include ssdp?

Sure. That effectively makes it an alias.

**Comment by erig0** [comment]
> https://github.com/firewalld/firewalld/blob/c11510e065fdbaeb8ff20f32e2628bdd70d30dd7/config/services/upnp-client.xml#L5
> 
> btw this is wrong, because the source port may be random for inbound traffic (M-SEARCH, NOTIFY).

Correct. It was wrong. `upnp-client` probably never worked. Having it include `ssdp` seems like the correct thing to do.

**Review by erig0 — APPROVED** [review]
Thanks!

**Pull request merged**